src/kernel.h

```diff
@@ -24,6 +24,8 @@ bool IsProtocolV03(unsigned int nTimeCoinStake);
 bool IsProtocolV04(unsigned int nTimeBlock);
 // Whether a given transaction is subject to new v0.5 protocol
 bool IsProtocolV05(unsigned int nTimeTx);
+// Whether a given block or transaction is subject to new v0.6 protocol
+bool IsProtocolV06(unsigned int nTimeTx);
```

Comment:
For v0.5 it's only supposed to be used for coinstake transactions I think, whose nTime cannot be picked arbitrarily